tests/cel/v1beta3/main_test.go

```
@@ -0,0 +1,107 @@
+
+	rc := m.Run()
+	if testEnv != nil {
+		if err := testEnv.Stop(); err != nil {
+			panic(fmt.Sprintf("error stopping test environment: %v", err))
+		}
+	}
+
+	os.Exit(rc)
+}
+
+func ptrTo[T any](a T) *T {
```

golangci-lint (2.11.4)

[error] func ptrTo is unused

(unused)